tests/test_post_save.py

```diff
@@ -0,0 +1,83 @@
+import shutil
+
+import nbformat
+from notebook.services.contents.filemanager import FileContentsManager
+import pytest
+from traitlets.config import Config
+
+from nbautoexport.jupyter_config import initialize_post_save_hook
+from nbautoexport.sentinel import (
+    ExportFormat,
+    NbAutoexportConfig,
+    OrganizeBy,
+    SAVE_PROGRESS_INDICATOR_FILE,
+)
+
+
+@pytest.fixture()
+def notebook_file(tmp_path, notebook_asset):
+    nb_file = tmp_path / "the_notebook.ipynb"
+    shutil.copy(notebook_asset.path, nb_file)
+    return nb_file
+
+
+@pytest.fixture()
+def notebook_model(notebook_file):
+    model = {
+        "type": "notebook",
+        "content": nbformat.read(str(notebook_file), as_version=nbformat.NO_CONVERT),
+    }
+    return model
+
+
+@pytest.fixture()
+def file_contents_manager(notebook_file):
+    config = Config(FileContentsManager=FileContentsManager())
+    config.FileContentsManager.root_dir = str(notebook_file.parent)
+    initialize_post_save_hook(config)
+    return config.FileContentsManager
+
+
+def test_post_save(file_contents_manager, notebook_file, notebook_model):
+    """Test that post_save function works when FileContentsManager saves."""
+    config = NbAutoexportConfig(
```

Comment:
Is it worth parameterizing this test to check the saves work for each of the supported formats? Currently this just tests the default.

Reply:
I added a second step to the that test that checks that it writes different export files when the config file is updated, which is something your comment highlighted that I wasn't covering before. I don't think it's necessary to parameterize all options—the `test_export.py` module already tests all of that pretty aggressively.

Reply:
Nice -- I thought we had those exports tested elsewhere but had only seen the cli exports on my skim. Thanks!